codegen-core/src/test/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/EnumGeneratorTest.kt

```diff
@@ -139,12 +138,13 @@ class EnumGeneratorTest {
                     assert_eq!(InstanceType::from("t2.nano"), InstanceType::T2Nano);
                     """,
                 )
-                val output = toString()
-                output.shouldContain("#[non_exhaustive]")
-                // on enum variant `T2Micro`
-                output.shouldContain("#[deprecated]")
-                // on enum itself
-                output.shouldContain("#[deprecated(since = \"1.2.3\")]")
+                toString().also { output ->
+                    output.shouldContain("#[non_exhaustive]")
+                    // on enum variant `T2Micro`
+                    output.shouldContain("#[deprecated]")
+                    // on enum itself
+                    output.shouldContain("#[deprecated(since = \"1.2.3\")]")
```

Comment:
It would be kind of neat if you could do:
```suggestion
                    output.shouldContain(Attribute.NonExhaustive)
                    // on enum variant `T2Micro`
                    output.shouldContain(Attribute.Deprecated())
                    // on enum itself
                    output.shouldContain(Attribute.deprecated(since = "1.2.3"))
```

you'd have to overload `shouldContain` though